### DISCUSSION TITLE
How to view css of the codeblock component in viewer UI?

### DISCUSSION THREAD

**Discussion by TungNguyen-C4U** [hunghg255/reactjs-tiptap-editor#129] [Q&A] [answered]
Hi @hunghg255 
I know your codeblock component is using Shiki package which I don't know how to show the highlight code (CSS of it) in the viewer UI since I can not find it anywhere in Shiki source. Can you help me how to view css of the codeblock component in viewer UI?

Thank you for your help

**Comment by hunghg255** [comment, marked as answer]
> Hi @hunghg255 I know your codeblock component is using Shiki package which I don't know how to show the highlight code (CSS of it) in the viewer UI since I can not find it anywhere in Shiki source. Can you help me how to view css of the codeblock component in viewer UI?
> 
> Thank you for your help

you can try this https://prismjs.com/
